runatlantis.io/docs/custom-workflows.md

```diff
@@ -407,3 +407,24 @@ as the environment variable value.
 * `env` `command`'s can use any of the built-in environment variables available
   to `run` commands. 
 :::
+
+#### Multiple Environment Variables `multienv` Command
+The `multienv` command allows you to set dynamic number of multiple environment variables that will be available
+to all steps defined **below** the `multienv` step.
+```yaml
+- multienv: custom-command
+```
+| Key      | Type   | Default | Required | Description                                   |
+|----------|--------|---------|----------|-----------------------------------------------|
+| multienv | string | none    | no       | Run a custom command and add set              |
+|          |        |         |          | environment variables according to the result |
+The result of the executed command must have a fixed format:
+EnvVar1Name=value1,EnvVar2Name=value2,EnvVar3Name=value3
+
+The name-value pairs in the result are getting added as envinronment variables if success is true otherwise the workflow execution stops with error and the errorMessage is getting displayed.
```

Comment:
```suggestion
The name-value pairs in the result are added as environment variables if success is true otherwise the workflow execution stops with error and the errorMessage is getting displayed.
```